platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -366,4 +424,67 @@ private int findJoystickDevice(int device_id) {
 
 		return -1;
 	}
+
+	private boolean handleMouseEvent(final MotionEvent event) {
```

Comment:
Can you add more documentation comments in this method since it's fairly long and does a lot.

Reply:
Im not sure how, it's pretty self explenitory from the code by looking at the cases